src/templates/howto.html

```diff
@@ -105,7 +103,7 @@ <h3>Countries</h3>
 			<li>Looks like we cannot load the list of supported countries from our server. That's embarrassing. Sorry.</li>
 		</ul>
 		<p class="buttons">
-			<a class="purple button" href="/start/">Set up a swap</a>
+			<a class="purple button" href="/start/" style="color: white;">Set up a swap</a>
```

Comment:
Did you meant to add this `style="color: white;"` part? If not, you might have to pull from upstream first.
(not sure how to do that in GitHub Desktop, sorry. Jaram will probably be able to help...)